sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -0,0 +1,911 @@
+         4
+        :keyword index_filter: A comma separated list of indexes which should be utilized for the search.
+         Item order does not matter.
+        :paramtype index_filter: list[str or ~azure.maps.search.models.SearchIndexes]
+        :keyword list[str] brand_filter: A comma-separated list of brand names which could be used to restrict the
+         result to specific brands. Item order does not matter. When multiple brands are provided, only
+         results that belong to (at least) one of the provided list will be returned. Brands that
+         contain a "," in their name should be put into quotes.
+        :keyword electric_vehicle_connector_filter: A comma-separated list of connector types which could
+         be used to restrict the result to Electric Vehicle Station supporting specific connector types.
+         Item order does not matter. When multiple connector types are provided, only results that
+         belong to (at least) one of the provided list will be returned.
+        :paramtype electric_vehicle_connector_filter: list[str or ~azure.maps.search.models.ElectricVehicleConnector]
+        :keyword entity_type: Specifies the level of filtering performed on geographies. Narrows the
+         search for specified geography entity types, e.g. return only municipality. The resulting
+         response will contain the geography ID as well as the entity type matched. If you provide more
+         than one entity as a comma separated list, endpoint will return the 'smallest entity
+         available'.
+        :paramtype entity_type: str or ~azure.maps.search.models.GeographicEntityType
+        :keyword localized_map_view: The View parameter (also called the "user region" parameter) allows
+         you to show the correct maps for a certain country/region for geopolitically disputed regions.
+         Different countries have different views of such regions, and the View parameter allows your
+         application to comply with the view required by the country your application will be serving.
+         By default, the View parameter is set to “Unified” even if you haven’t defined it in  the
+         request. It is your responsibility to determine the location of your users, and then set the
+         View parameter correctly for that location. Alternatively, you have the option to set
+         ‘View=Auto’, which will return the map data based on the IP  address of the request.
+        :paramtype localized_map_view: str or ~azure.maps.search.models.LocalizedMapView
+        :keyword operating_hours: Hours of operation for a POI (Points of Interest). The availability of
+         hours of operation will vary based on the data available. If not passed, then no opening hours
+         information will be returned.
+         Supported value: nextSevenDays.
+        :paramtype operating_hours: str or ~azure.maps.search.models.OperatingHoursRange
+        :return: SearchAddressResult, or the result of cls(response)
```

Comment:
Please remove any reference to `"...or the result of cls(response)"` from the public docstrings - this is for internal use only.
Additionally please add the `:rtype:` tags to all of the docstrings.
```python
:returns: The results of the search.
:rtype: ~azure.maps.search.models.SearchAddressResult
```